### ISSUE TITLE
Mock meeting info page

### ISSUE THREAD

**Issue by CraigBryan** [CraigBryan/meeto-mock#8]

**Comment by CraigBryan** [comment]
I've pushed a partial implementation. The discussion part is empty, but that's because I'll try and style it like the messages page, so I wait until that's done.

**Comment by DanielMurdoch** [comment]
I added the links on other pages so that they properly go to meeting.html,
like when you sign up or from the scheduled meetings page

**Comment by CraigBryan** [comment]
Great. I assume that's something we'll have to go over again when everything is done, the mapping of links to pages.

**Comment by DanielMurdoch** [comment]
Yeah exactly, hence why I'm trying to go through the links I can remember
now so that none are missed later aha,

**Comment by DanielMurdoch** [comment]
Messages seems to be done, should the discussion look the same as that? I think that's pretty fine.

**Comment by CraigBryan** [comment]
I see the list of discussions page, but not the actual messages between people. That's what needs to go in the discussions page... the actual messages.

**Comment by DanielMurdoch** [comment]
Truuuuue. Yeah i dunno then. I dunno if Ryan was intending on doing that or not.

**Comment by charle692** [comment]
Yeah I'm working on the actual discussions part of it. Making it like we
decided in the storyboards.

**Comment by charle692** [comment]
Ok I've completed the discussions page.

**Comment by DanielMurdoch** [comment]
Think we can plug that into the meetings page pretty easily?
